docs/features/shortcuts/README.md

```diff
@@ -80,6 +80,13 @@ The list below contains available keyboard shortcuts grouped by problem areas. Y
 * <kbd>Shift+PgUp</kbd> &ndash; Selects a text fragment of approximately the length of the editing area starting from the cursor
 	and going up.
 
+### On widget selection
```

Comment:
```suggestion
### Widget Selection
```

:point_up: Let's make it consistent with the rest of the headings here:

![](https://user-images.githubusercontent.com/1061942/116672503-8cfb2f00-a9a2-11eb-8984-b3c9f0b119ab.png)